lib/.eslintrc

```diff
@@ -0,0 +1,13 @@
+{
+  "parserOptions": {
+      "ecmaVersion": 5
+  },
+  "extends": "eslint:recommended",
+  "rules": {
+    "no-undef": "warn",
```

Comment:
What would be the best way to deal with js_of_ocaml functions like `caml_js_to_array` that are available globally? Should we add them to some global file so eslint ignores them?

Reply:
Added global variables:
https://github.com/jchavarri/rroo/pull/20/files#diff-b0d0bf091f28af6d677bffef8898431bR10

I turned the `"no-undef"` rule to `"error"`, so when we run eslint next time we add new global variables we will know to add it to the `.eslintrc`.